rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n3.yaml

```diff
@@ -0,0 +1,25 @@
+---
+###############################################################################
+# SR-IOV Network Definition for the NS of the workload
+#  N3-bh + vlan 904 + core_sriov_netdevice
+###############################################################################
+# NS = open5gs-core
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: n3bh
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 904
```

Comment:
vlan 904 is also not defined anywhere in the lab

Reply:
Same argument as above for VLANs.